packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -109,11 +109,7 @@ const useRbfState = ({ tx, finalize, chainedTxs }: Props, currentState: boolean)
     let { baseFee } = tx.rbfParams;
     if (chainedTxs.length > 0) {
         // increase baseFee, pay for all child chained transactions
-        baseFee = chainedTxs.reduce((f, ctx) => {
-            // TODO: transformation to satoshi should not be here, refactor of "enhanceTransaction" required
```

Comment:
nice to see that this transaction enhancement was causing problems on other places too :+1: and it is gone now